wsgi.py

```diff
@@ -5,13 +5,13 @@
 
 dotenv.load_dotenv()
 
-logging.basicConfig(filename='dashi.log', level=logging.INFO)
+logging.basicConfig(filename=(os.getenv("LOG_FILE_LOCATION") or "dashi.log"), level=logging.INFO)
```

Comment:
README has the default location at `~/logs/dashi.log`, but this would save it where the Dashi app was started.

Reply:
The README indicates an example location, not a default location. I can update the README to indicate the default location as well, and change this to match that default location.

Reply:
Updated the README to indicate that the file is created in the directory where the app was started (as I didn't want to make any assumptions about file systems and `~/`...)